code/index.html

```diff
@@ -3,28 +3,182 @@
   <head>
     <meta charset="utf-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <title>Project 3</title>
+    <title>Project 3: Business site - Balloon ride</title>
+    <link rel="stylesheet" href="./style.css" />
+    <style>
+      @import url('https://fonts.googleapis.com/css2?family=Lobster&family=Roboto+Condensed:wght@300&display=swap');
+    </style>
```

Comment:
When you're using @import for fonts they should be places at the top of the CSS file instead. In the HTML we can use the link import instead.